src/containers/DataGUI.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
Why are we duplicating this? We already have `Metafields` component that does the same thing.